xwiki-rendering-macros/xwiki-rendering-macro-message/src/main/java/org/xwiki/rendering/internal/macro/message/MacroIconPrettyNameProvider.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.rendering.internal.macro.message;
+
+import org.xwiki.component.annotation.Component;
+
+import javax.inject.Singleton;
+import java.util.Map;
+
+/**
+ * Default component to provide an alternative text for an icon.
+ *
+ * @version $Id$
+ * @since 16.7.0RC1
+ */
+@Singleton
+@Component(roles = MacroIconPrettyNameProvider.class)
+public class MacroIconPrettyNameProvider
+{
+    private static final Map<String, String> PRETTYNAME_MAPPING = Map.of(
+        "error", "Error",
+        "info", "Information",
+        "success", "Success",
+        "warning", "Warning"
+    );
+
+    /**
+     * @param macroId the id of the macro whose icon needs an alternative text
+     * @return the alternative text associated to the provided icon
+     */
+    String getIconPrettyName(String macroId) 
+    {
+        return PRETTYNAME_MAPPING.get(macroId);
```

Comment:
I'm not sure it really worth having such a mapping. What about `StringUrils.capitalize(macroId)` ? Feels good enough for a fallback we never really use in practice.

Reply:
I went with this solution first, because it's usually bad for accessibility to use programmatic names as descriptions. Here it does not really make a difference so might as well use the macro Id directly :+1: 
Addressed in https://github.com/xwiki/xwiki-rendering/pull/296/commits/75253ad3a2ea2d5e670085a8b024c0a93f35e392

Reply:
Addressed in https://github.com/xwiki/xwiki-rendering/pull/296/commits/75253ad3a2ea2d5e670085a8b024c0a93f35e392 :+1: